tests/baselines/reference/nodeModulesTripleSlashReferenceModeDeclarationEmit3(module=node16).js

```diff
@@ -25,15 +25,15 @@ declare global {
     "type": "module"
 }
 //// [index.ts]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" resolution-mode="require" preserve="true" />
```

Comment:
One of many examples of `preserve` appearing; maybe we should just not write these to dts files? We already selectively omit `resolution-mode` when it's redundant to the implied format of the module, so it feels like dropping `preserve` would be fine.

Reply:
The answer from the design meeting was yes, keep it.